Applications/Classification/jni/main_func.cpp

```diff
@@ -0,0 +1,305 @@
+/**
+ * Copyright (C) 2019 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Q. Is the spdx license idendifier, /* SPDX-License-Identifier: Apache-2.0 */, used only for header files?

Reply:
As far as I know, SPDX is short term for this paragraph. So we could choose either one of them. 
And I think we could use it in source file as well. Correct me if I'm wrong.